stable/traefik/templates/service.yaml

```diff
@@ -34,12 +34,21 @@ spec:
   ports:
   - port: 80
     name: http
+    {{- if and (eq .Values.serviceType "NodePort") (.Values.nodePort.httpPort) }}
```

Comment:
Hi @sean-brandt If you want to add a `nodePort` option, you would also need to:
- Define default value in `values.yaml` with comments explaining when to use it (initially commented out to preserve current functionality).
- Update `NOTES.txt` explaining how to get the URL if `serviceType` is `nodePort` (see [stable/jenkins NOTES.txt](https://github.com/kubernetes/charts/blob/master/stable/jenkins/templates/NOTES.txt) for an example of how to do this).

Another suggestion: given the current organization of `values.yaml` this chart, it may be good to add this variable below the existing `service` key (`services.nodePort`), as some other charts do. There isn't a current standard about how values are organized, so this one is just an opinion.

Reply:
Agree with everything @scottrigby said. Do you have some time to update this @sean-brandt?